modules/ROOT/pages/migrate-ah.adoc

```diff
@@ -0,0 +1,44 @@
+:experimental:
+
+= Migrating from Atomic Host (AH) to Fedora CoreOS (FCOS)
+
+== Overview
+
+https://www.projectatomic.io/[Atomic Host] is a system to deploy applications in containers. Atomic Hosts users are encouraged to migrate to FCOS, as the project is being folded into FCOS.
+
+Atomic Host uses a cloud-config file to define parameters of the containers. There is currently no tool to migrate cloud-config files to FCOS Ignition files. The conversion is not direct, as some parameters in Ignition are located in different Atomic Host files.
+
+== Examples
+
+=== Converting `user-data`
+
+An Atomic Host `user-data` file looks like the following:
```

Comment:
we should probably call it cloud-init cloud-config userdata to bring the terminology forward from the previous sections